USEFUL LINKS

Saccharomyces Genome Database (SGD): (https://www.yeastgenome.org/)

YESDB: integrative analysis of environmental stress in yeast (https://services.bio.ifi.lmu.de:1047/YESdb/)

Yeast Nucleosome Atlas (YNA): (http://cosbi3.ee.ncku.edu.tw/yna/statistic)

Micro-C Nucleosome Mapping: (https://pubmed.ncbi.nlm.nih.gov/26119342/)

High-temporal-resolution view of transcription and chromatin states across distinct metabolic states in budding yeast: (https://pubmed.ncbi.nlm.nih.gov/25173176/)

Genetic dissection of complex traits using hierarchical biological knowledge: (https://hgdownload.soe.ucsc.edu/goldenPath/sacCer3/multiz7way/fasta/)

ChIP-Seq Atlas Database: (https://chip-atlas.org/)

Learning the histone codes with large genomic windows and three-dimensional chromatin interactions using transformer: (https://pubmed.ncbi.nlm.nih.gov/36335101/)

Mus musculus (house mouse) Gene Expression in d5 wound-edge tissues of MFG-E8 WT and MFG-E8 KO mice: (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA296581 downloadable here https://www.ncbi.nlm.nih.gov/geo/info/download.html

Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks: (https://ega-archive.org/datasets/EGAD00001003824)

Temporal transcriptional logic of dynamic regulatory networks underlying nitrogen signaling and use in plants: (https://datadryad.org/stash/dataset/doi:10.5061/dryad.248g184)

Visualize and interactively explore bio-human-gene1 and its important node-level statistics! (https://networkrepository.com/bio-human-gene1.php)

Target genes of transcription factors from transcription factor binding site profiles: (https://maayanlab.cloud/Harmonizome/dataset/ENCODE+Transcription+Factor+Targets)

More from Google Datasets: (https://datasetsearch.research.google.com/search?ref=TDJjdk1URnFibmwyWTNGZlh3PT0sTDJjdk1URnFZMnB5Y1hoc2F3PT0sTDJjdk1URnlhMkk0ZG1JMmVBPT0%3D&query=gene%20regulation%20edges&docid=L2cvMTFqbnl2Y3FfXw%3D%3D&filters=bm9uZQ%3D%3D&property=bGljZW5zZV9jbGFzcw%3D%3D)
